setup.py

```diff
@@ -1,22 +1,24 @@
-""" podpac module"""
-
 # Always perfer setuptools over distutils
 import sys
-from setuptools import setup, find_packages
+
+from setuptools import find_packages, setup
+
+import version
+
+""" podpac module"""
+
 
 # get version
 sys.path.insert(0, 'podpac')
-import version
```

Comment:
`import version` has to go below this line.

Reply:
Just pushed this updated, thanks!